.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: php
+
 php:
-  - 5.6
-  - 5.5
-  - 5.4
   - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
   - 7.0
```

Comment:
Could all of the versions above be removed so that we're just left with `- hhvm`? Won't all others be ran as a result of the matrix?